packages/backend-dynamic-feature-service/src/scanner/plugin-scanner.ts

```diff
@@ -68,7 +70,7 @@ export class PluginScanner {
   }
 
   private applyConfig(): void | never {
-    const dynamicPlugins = this.config.getOptional('dynamicPlugins');
+    const dynamicPlugins = this.config.getOptional(configKey);
     if (!dynamicPlugins) {
       this.logger.info("'dynamicPlugins' config entry not found.");
```

Comment:
```suggestion
      this.logger.info(`'${configKey}' config entry not found.`);
```

and the same in all errors below etc

Reply:
Fixed